lib/db/models/Node.ts

```diff
@@ -20,6 +20,19 @@ export default (sequelize: Sequelize.Sequelize, DataTypes: Sequelize.DataTypes)
         }
       },
     },
+    lastAddressText: { type: Sequelize.TEXT, allowNull: true },
+    lastAddress: {
+      type: Sequelize.VIRTUAL,
+      allowNull: true,
+      get(this: db.NodeInstance) {
+        return this.lastAddressText ? JSON.parse(this.lastAddressText) : undefined;
+      },
+      set(this: db.NodeInstance, value: Address) {
+        if (value) {
+          this.setDataValue('lastAddressText', JSON.stringify(value));
```

Comment:
Can we `setDataValue` to `null` or `undefined` if `value` is `undefined`?

Reply:
Actually not setting it wouldn't be sufficient, isn't it actually the same thing with setting is as undefined since we already defined it as `lastAddress?` ? Also trying to set undefined to a `TEXT` field is not allowed I guess

Reply:
OK, I wasn't too sure.  Thanks.